main.go

```diff
@@ -63,12 +66,19 @@ func main() {
 		exit(fmt.Sprintf("--since must start before --until. --since: %s, --until: %s", *since, *until))
 	}
 
-	incidents, err := fetchIncidents(*team, sinceAt, untilAt)
-	if err != nil {
-		exit("Failed to fetch incidents from Datadog: %v", err)
+	var incidents []*incident
+	for _, team := range *teams {
+		teamIncidents, err := fetchIncidents(team, sinceAt, untilAt)
+		if err != nil {
+			exit("Failed to fetch incidents from Datadog: %v", err)
+		}
+		incidents = append(incidents, teamIncidents...)
```

Comment:
Should we instead change the filter in `fetchIncidents` to grab multiple teams, I haven't touched this code in a while but should be possible? cc @badovm @robertjli